styles.css

```diff
@@ -8,6 +10,18 @@ body {
   grid-gap: 1px;
 }
 
+h2 {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+input {
+  border-color: #5E60A9;
+  border-radius: 2px;
+}
+
+/* IDEABOX RULES (SIDEBAR) */
```

Comment:
I love that each section is categorized!

Reply:
Looks great!